packages/react-server-cli/src/parseCliArgs.js

```diff
@@ -2,6 +2,59 @@ import yargs from "yargs"
 import fs from "fs"
 import pem from "pem"
 
+const sslize = async argv => {
```

Comment:
Oh, heh, you had the [same thought](https://github.com/redfin/react-server/pull/770#discussion_r89396334). 😁